src/services/compare/handlers/sendDoesNotExistAlert.js

```diff
@@ -14,7 +21,7 @@ exports.handler = async function (event, context, callback) {
   const stage = process.env.stage;
 
   // use this secret path to define the { emailRecipients, sourceEmail } for the does not exist email
-  const secretId = `${project}/${stage}/alerts`;
+  let secretId = `${project}/${stage}/alerts`;
```

Comment:
honestly I would try and see if we can get away with just using one secret. And since we will have another comparison service (appian) we may want to just use one secret named something like: `${project}/${stage}/mmdl-alerts` and then keep both CHP and non-CHP recipient information in there. 

this way we can avoid having 4 secrets per higher environment.